Tasks/Malcheuskaya/task4.py

```diff
@@ -0,0 +1,77 @@
+while True:
+    maxNum = input("Enter maximum number of characters per line (over 35): ")
+    if maxNum == "":                    
+        break
+    if not maxNum.isdigit(): 
+        print("Error: enter only number!")
+        continue
+    maxNum = int(maxNum) 
+    if maxNum <= 35:
+        print("Error: enter number over 35!")
+        continue
+    try:
+        with open("text.txt", "r") as file:
+            textList =[] 
+            for i in file.read().replace("вЂ™", "'").replace('вЂњ', '"').replace('вЂќ', '"').split("\n"): 
+                lineList = [] 
+                line = "" 
+                for j in i.split(" "): 
+                    if len(line + j) <= maxNum: 
+                        line += j + " " 
+                    else:
+                        lineList.append(line.rstrip()) 
+                        line = j + " " 
+                lineList.append(line.rstrip()) 
+                j = 0 
+                while j < len(lineList) - 1: 
+                    numOfSpaces = int((maxNum - len(lineList[j]))/lineList[j].count(" ")) + 1 #Вычисления минимальной длины пробелов для выравнивания строки
+                    lineList[j] = lineList[j].replace(" ", " " * numOfSpaces) #Увеличение всех пробелов строки на минимальное значение
+                    lineList[j] = lineList[j].replace(" " * numOfSpaces, " " * (numOfSpaces + 1), maxNum - len(lineList[j])) #увеличение пробелов для окончательного выравнивания
+                    j += 1 
+                textList.append(lineList) 
+    except:
```

Comment:
it's a generic except, so it can intercept any error, but in any case you will notify user that the file is missing, which could be not true